introduction.md

```diff
@@ -0,0 +1 @@
+Hi I'm Amy and I'm here to step up my git knowledge :)
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.